src/main/java/dev/joopie/jambot/music/command/PlayCommandHandler.java

```diff
@@ -90,8 +90,9 @@ public RestAction<?> handle(final CommandInteraction event) {
 
             if (input.contains(SPOTIFY)) {
                 return handlePlayWithInteractions(event, handleSpotifyLink(input));
-            } else if (input.contains("youtube") || input.contains("youtu.be")){
-                return handlePlay(event, extractYouTubeVideoId(input));
+            } else if (input.contains("youtube") || input.contains("youtu.be")) {
+                var extractedId = extractYouTubeVideoId(input);
+                return handlePlay(event, extractedId != null ? extractedId : Strings.EMPTY);
```

Comment:
String empty wordt alsnog meegeven aan de musicService. Dit lijkt mij niet wenselijk. Wat gebeurt er dan?

Als er geen video id gextract kan worden, stoppen en melding teruggeven dat niet is gelukt ofzo.

Reply:
De `handleSpotifyLink` daarboven returnt ook een Strings.EMPTY. Zal deze ook meenemen.